colcon_test_result/verb/test_result.py

```diff
@@ -109,11 +109,11 @@ def main(self, *, context):  # noqa: D102
         return 1 if summary.error_count or summary.failure_count else 0
 
 
-def _argparse_existing_dir(path):
+def _verify_dir_exists(path):
     if not os.path.exists(path):
-        raise argparse.ArgumentTypeError("Path '%s' does not exist" % path)
+        raise RuntimeError("Path '%s' does not exist" % path)
```

Comment:
Changed to a RuntimeError as its a "known" error
https://github.com/colcon/colcon-core/blob/master/colcon_core/command.py#L523